src/util/misc.ts

```diff
@@ -173,25 +173,25 @@ export function saveStaticFiles(config: Config, zimCreator: ZimCreator) {
   ]);
 }
 
-export function cssPath({ output: { dirs } }: Config, css: string) {
-  return [dirs.style, `${dirs.styleModules}/${css.replace(/(\.css)?$/, '')}.css`].join('/');
+export function cssPath(css: string, subDirectory = '') {
```

Comment:
We should maybe add type checking here for `subDirectory`?

Reply:
ok will add that as well